docs/install.md

```diff
@@ -0,0 +1,70 @@
+# Installing the Knative Pipelines component
+
+Before you can run Knative Pipelines, you must install the Knative Pipelines components
+in your Kubernetes cluster. Use this page to add the component to an existing 
+Kubernetes cluster.
+
+## Before you begin
+
+For complete installation instructions of Knative components, see [Installing Knative](https://github.com/knative/docs/blob/master/install/README.md).
+
+## Adding the Knative Pipelines component
+
+To add only the Knative Pipelines component to an existing installation:
+
+1. Run the
+   [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply)
+   command to install [Knative Pipelines](https://github.com/knative/build-pipeline) and its
+   dependencies:
+   ```bash
+   kubectl apply --filename https://storage.googleapis.com/knative-releases/build-pipeline/latest/release.yaml
```

Comment:
could we mention something here about installing [versioned releases](https://github.com/knative/build-pipeline/blob/master/hack/release.md#creating-versioned-releases), even if we don't have them yet? or would you rather wait until we have one XD

Reply:
I prefer to wait till there is at least one previous release so we don't confuse users, wdyt?

Reply:
kk, sounds good!

Reply:
hm actually it looks like we're planning to hold off on merging this until we have a real release (good point @shashwathi XD), so in that case, we could update it pre-emptively with a reference to versioned releases (we'd be starting with 0.1)

Also wondering if we want the path to be `knative-releases/build-pipeline` if we're going to decouple our release from `knative` :thinking:

Reply:
+ @dlorenc @ImJasonH

Reply:
I'm not sure I understand what you want to add about installing `versioned releases`, don't we want users to install latest? The link you mention in the comment here is about creating a new versioned release

Reply:
> I'm not sure I understand what you want to add about installing `versioned releases`, don't we want users to install latest?

Some users will want latest, but some will want to pin to an explicit version